website/project/views/contributor.py

```diff
@@ -505,6 +507,29 @@ def send_claim_email(email, user, node, notify=True, throttle=24 * 3600):
     return to_addr
 
 
+@contributor_added.connect
+def notify_contributor(node, contributor, throttle=24 * 3600):
```

Comment:
`notify_contributor` is an ambiguous name; we notify contributors about a number of things. How about `notify_added_contributor`?